lib/draw/BpmnRenderer.js

```diff
@@ -852,445 +719,640 @@ export default function BpmnRenderer(
+        strokeWidth: 2,
+        fill,
+        stroke
       });
+    }
 
-      return task;
-    },
-    'bpmn:SubProcess': function(parentGfx, element, attrs) {
-      attrs = {
-        fill: getFillColor(element, defaultFillColor),
-        stroke: getStrokeColor(element, defaultStrokeColor),
-        ...attrs
-      };
+    return dataObject;
+  }
+
+  function renderEvent(parentGfx, element, attrs = {}) {
+    return drawCircle(parentGfx, element.width, element.height, {
+      fillOpacity: DEFAULT_OPACITY,
+      ...attrs,
+      fill: getFillColor(element, defaultFillColor, attrs.fill),
+      stroke: getStrokeColor(element, defaultStrokeColor, attrs.stroke)
+    });
+  }
 
-      var rect = renderer('bpmn:Activity')(parentGfx, element, attrs);
+  function renderGateway(parentGfx, element, attrs = {}) {
+    return drawDiamond(parentGfx, element.width, element.height, {
+      fill: getFillColor(element, defaultFillColor, attrs.fill),
+      fillOpacity: DEFAULT_OPACITY,
+      stroke: getStrokeColor(element, defaultStrokeColor, attrs.stroke)
+    });
+  }
 
-      var expanded = isExpanded(element);
+  function renderLane(parentGfx, element, attrs = {}) {
+    var lane = drawRect(parentGfx, getWidth(element, attrs), getHeight(element, attrs), 0, {
+      fill: getFillColor(element, defaultFillColor, attrs.fill),
+      fillOpacity: attrs.fillOpacity || DEFAULT_OPACITY,
+      stroke: getStrokeColor(element, defaultStrokeColor, attrs.stroke),
+      strokeWidth: 1.5
+    });
 
-      if (isEventSubProcess(element)) {
-        svgAttr(rect, {
-          strokeDasharray: '0, 5.5',
-          strokeWidth: 2.5
-        });
-      }
+    var semantic = getSemantic(element);
 
-      renderEmbeddedLabel(parentGfx, element, expanded ? 'center-top' : 'center-middle');
+    if (is(semantic, 'bpmn:Lane')) {
+      var text = semantic.get('name');
 
-      if (expanded) {
-        attachTaskMarkers(parentGfx, element);
-      } else {
-        attachTaskMarkers(parentGfx, element, [ 'SubProcessMarker' ]);
-      }
+      renderLaneLabel(parentGfx, text, element, attrs);
+    }
 
-      return rect;
-    },
-    'bpmn:AdHocSubProcess': function(parentGfx, element) {
-      return renderer('bpmn:SubProcess')(parentGfx, element);
-    },
-    'bpmn:Transaction': function(parentGfx, element) {
-      var outer = renderer('bpmn:SubProcess')(parentGfx, element, { strokeWidth: 1.5 });
+    return lane;
+  }
 
-      var innerAttrs = styles.style([ 'no-fill', 'no-events' ], {
-        stroke: getStrokeColor(element, defaultStrokeColor),
-        strokeWidth: 1.5
-      });
+  function renderSubProcess(parentGfx, element, attrs = {}) {
+    var expanded = isExpanded(element);
 
-      /* inner path */ drawRect(parentGfx, element.width, element.height, TASK_BORDER_RADIUS - 3, INNER_OUTER_DIST, innerAttrs);
+    if (!expanded) {
+      attrs = pickAttrs(attrs, [
+        'fill',
+        'stroke'
+      ]);
+    }
```

Comment:
`strokeWidth` is passed as an attribute into `renderSubProcess`, but it is not picked from the attributes here

Reply:
I broke this last minute. 🙈 Makes me think I should try adding a snapshot test.